### ISSUE TITLE
Uninstalled flash, youtube doesn't work

### ISSUE THREAD

**Issue by sapphir8** [hoyois/clicktoplugin#166]
I finally got tired of flash and uninstalled it from my Mac (Safari 9.1, OS X 10.11.4).  Now when I go to play a youtube video, it's just a black screen and nothing plays.  As soon as I disable clicktoplugin, it works fine.  How odd.  Any ideas?

**Comment by vmachiel** [comment]
Same issue. It has been a while since your post, did you get it working again?

**Comment by sapphir8** [comment]
Nope.

On Jul 27, 2016, at 11:04 AM, vmachiel <notifications@github.com<mailto:notifications@github.com>> wrote:

Same issue. It has been a while since your post, did you get it working again?

##

**Comment by SlapMyFanBoy** [comment]
I've never had flash installed yet clicktoplugin has worked fine for controlling youtube. As of today I get the black screen. No placeolder- no preview image - no source selector. I had to **install** flash for the killer to work. Blocking or allowing plug-ins in Safari prefs makes no difference. Now flash is running amok all over my browser windows. Sucky.

**Comment by vmachiel** [comment]
Yeah, ME TOO! Is there anything that could be done with the current version of the plugin, or is it over now? I like the CTF player so much better.

**Comment by sjkx** [comment]
See #171 for details.
